cart/cart.js

```diff
@@ -48,4 +48,5 @@ orderButton.addEventListener('click', () => {
     alert(stringyCart);
     localStorage.clear();
     window.location.href = '/';
```

Comment:
you may have to apply a more relative path for it to work on github.  maybe trying: '../index.html' would work for this one'

Reply:
Okay will do it after the presentations I just am confused why it shows up in my live but not on the pages